container/go/cmd/puller/puller.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The Bazel Authors. All rights reserved.
+/// Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
Try to avoid this kind of unnecessary change which is not really related to the purpose of this PR.